be/src/formats/json/boolean_column.cpp

```diff
@@ -0,0 +1,53 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "boolean_column.h"
+
+#include "column/binary_column.h"
+#include "gutil/strings/substitute.h"
+
+namespace starrocks::vectorized {
+
+static auto literal_0_slice{Slice{"0"}};
+static auto literal_1_slice{Slice{"1"}};
+
+Status add_boolean_column(Column* column, const TypeDescriptor& type_desc, const std::string& name,
+                          simdjson::ondemand::value& value) {
+    auto binary_column = down_cast<BinaryColumn*>(column);
```

Comment:
I thought it should be a `BooleanColumn`, why it's a `BinaryColumn`?

Reply:
Column in `TYPE_BOOLEAN` is treated as `TYPE_VARCHAR`, which is defined here:
https://github.com/StarRocks/starrocks/blob/0b1044bc5eaeaec1224a6bbde4644d97a3927fbf/be/src/exec/vectorized/json_scanner.cpp#L138